akka-persistence/src/test/scala/akka/persistence/PersistentActorStashingSpec.scala

```diff
@@ -194,6 +214,20 @@ abstract class PersistentActorStashingSpec(config: Config) extends PersistenceSp
     behave like stashUnderFailures[UserStashWithinHandlerFailureCallbackPersistentActor]()
   }
 
+  "Stashing(stash called in handler) in a persistent actor" must {
+    "fail when calling stash in persist handler" in {
+      val actor = namedPersistentActor[StashWithinHandlerPersistentActor]
+
+      actor ! Cmd("a")
+      expectMsg("a")
```

Comment:
This is a bit of an 'indirect' way of testing - would it be possible to let the actor crash and get the exception from the supervisor?

Reply:
Agree with @raboof; I'd propose to send back the exception to the test rather than the event, and assert on it's message here in the test rather than in the actor

Reply:
👍

Reply:
@ktoso "send back the exception to the test rather than the event" how can i return it back to the test. You meant via supervisor